src/containers/ListDetails/component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import {
+  Row, Col, Typography, Spin, Empty
+} from 'antd'
+import PropsTypes from 'prop-types'
+
+import MoviesList from 'Components/MoviesList'
+import { MinusCircleOutlined } from '@ant-design/icons'
+import DeleteMovie from './DeleteMovie'
+
+const ListDetails = ({
+  showDeleteListModal, movies, listName, loading, isBlank
+}) => (
+  <>
+    <Row>
+      <Col
+        offset={2}
+        span={20}
+      >
+        {listName && (
+          <div className="top-margin">
+            <Typography.Title>
+              {listName}
+              {' '}
+              <MinusCircleOutlined onClick={showDeleteListModal} />
```

Comment:
sometimes you use name pattern for the handlers, sometimes no. choose approach which you like more and follow it everywhere in the project